README.md

```diff
@@ -4,33 +4,84 @@
 [![Go Report Card][report-card-img]][report-card]
 [![License][license-img]][license]
 
-# harbor-scanner-trivy
+# Harbor Scanner Adapter for Trivy
 
-The Harbor Scanner Adapter for [Trivy][trivy-url].
-See [Pluggable Image Vulnerability Scanning Proposal][image-vulnerability-scanning-proposal] for more details.
+The Harbor Scanner Adapter for [Trivy][trivy-url] is a service that translates the [Harbor][harbor-url] scanning API
+into the Trivy commands and allows Harbor to utilize Trivy for providing vulnerability reports on images stored in
+Harbor registry as part of its vulnerability scan feature.
+
+> See [Pluggable Image Vulnerability Scanning Proposal][image-vulnerability-scanning-proposal] for more details.
 
 ## TOC
 
-- [Deploy to Kubernetes (minikube)](#deploy-to-kubernetes-minikube)
+- [Getting started](#getting-started)
+  - [Installation](#installation)
+- [Testing](#testing)
+- [Deployment](#deployment)
 - [Configuration](#configuration)
 - [Documentation](#documentation)
+- [Contributing](#contributing)
+- [License](#license)
+
+## Getting started
+
+These instructions will get you a copy of the adapter service up and running on your local machine for development
+and testing purposes. See [deployment](#deployment) for notes on how to deploy on a live system.
```

Comment:
Strictly I think the "Getting Started" instructions just say how to build and run the unit and integration tests. 

Do you maybe want to leave the minikube instructions or is that no longer useful?

Reply:
You're right. I've reverted instructions for running on minikube.